src/woo_search/conf/base.py

```diff
@@ -25,6 +26,7 @@
     # Project applications.
     "woo_search.accounts",
     "woo_search.api",
+    "woo_search.index",
```

Comment:
a single app is fine, but maybe the name `search` or `search_index` is a bit better instead of `index`